cmd/utils.go

```diff
@@ -471,7 +471,6 @@ func newCustomHTTPTransport(tlsConfig *tls.Config, dialTimeout time.Duration) fu
 		DialContext:           newCustomDialContext(dialTimeout, 15*time.Second),
 		MaxIdleConnsPerHost:   16,
 		MaxIdleConns:          16,
-		MaxConnsPerHost:       64, // This is used per drive/rpc host. More requests will block until free.
```

Comment:
is this part of this fix?

Reply:
no it will be removed @cesnietor sent as a different fix

Reply:
cool apart from that all LGTM